sgtm/artemis/README.md

```diff
@@ -0,0 +1,273 @@
+# Artemis
+
+Get documentes from [GCP Firestore](https://firebase.google.com/docs/firestore) 
+databases in real time for advanced tag firing, value bidding, audience creation, 
+and onsite personalisation while keeping sensitive data away from determined end 
+users and third parties.
+
+Use [Server Side Google Tag Manager (sGTM)](
+https://developers.google.com/tag-platform/tag-manager/server-side) with
+[Firestore](https://cloud.google.com/firestore), to pull in cloud-based and
+potentially sensitive data. This enables multiple use cases:
+
+- Conditional tag firing e.g. for new and returning customers even when 
+guest checkout has been used
+- Supplementing onsite data with customer data from cloud, e.g. segmentation 
+groups, previous purchase history, and much more
+- Generating advanced bidding metrics in conjunction with [Soteria]
+(https://github.com/google-marketing-solutions/gps_soteria) and [Phoebe]
+(https://github.com/google-marketing-solutions/gps-phoebe).
+- Realtime on-site personalisation enabled by passing data back to the browser
+using sGTM's advanced tag behaviour
+
+For examples of how to set-up sGTM for these use cases see [XXX BELOW](kahaksjhdkasjhdaksdhaksjhdkasjdhkasjd).
+
+> Tip 💡: Check out [the sGTM Pantheon](
+    https://github.com/google-marketing-solutions/gps-sgtm-pantheon/tree/main)
+for more tags for different use-cases.
+
+## Background
+First party data is becoming ever more important to digital marketing. Marketers
+who are able to effectively use the insights and knowledge they have about their
+customers have an advantage over those who do not use this data. Accessing data
+about customers that is not immediatly available on the website is usually
+difficult though and requires significant web development and involves exposing 
+data on the website. This means many digital marketing teams do not make use of 
+the extensive data their BI teams have on their customers
+
+This data is sensitive, therefore most clients would not like a determined user 
```

Comment:
I'm not sure that these two paragraphs make sense, given the use case is more about data augmentation for customer data, conditional triggers etc.